cluster/examples/kubernetes/ceph/object-ec.yaml

```diff
@@ -38,8 +38,8 @@ spec:
       # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
       # for more info: https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size
       #target_size_ratio: ".5"
-  # Whether to preserve metadata and data pools on object store deletion
-  preservePoolsOnDelete: true
+  # Whether to preserve filesystem after CephFilesystem CRD deletion
+  preserveFilesystemOnDelete: true
```

Comment:
As mentioned above, `preservePoolsOnDelete` should not be changed for object or pool CRs, I must have overlooked this before

Reply:
Whoops clearly my find and replace overreached a bit. Reset back.